weblab/experiments/migrations/0027_experimentversion_experiment_key.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.23 on 2020-02-12 17:00
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Yep, I think this would be the way forward. Rather than trying to create new automatic migrations, it might actually be easier to edit the current files by hand.